functions/Test-DbaLastBackup.ps1

```diff
@@ -477,7 +487,7 @@ function Test-DbaLastBackup {
                                 Write-Message -Level Verbose -Message "Starting DBCC."
 
                                 $startDbcc = Get-Date
-                                $dbccresult = Start-DbccCheck -Server $destserver -DbName $dbName 3>$null
+                                $dbccresult = Start-DbccCheck -Server $destserver -DbName $dbName -MaxDop $MaxDop 3>$null
```

Comment:
This may just be me, but since they are using this primarily to get CHECKDB to be more efficient should we include this as a property of the output object? (maybe `DbccMaxdop`?)

Reply:
The output currently shows the full DBCC command executed.  I did not have to change anything.

Reply:
See the screenshot in the PR.

Reply:
Yes, I'm aware of that one. I'm referring to including it in a property by itself as it can pertain to the timing and having it as a property would allow for easier reporting if admins wanted that bit of info.

Reply:
I am not sure I follow about the property.  What would you add?
`RestoreElapsed = $restoreElapsed`
`DbccMaxDop     = $MaxDop`
`DbccStart      = [dbadatetime]$startDbcc`
`DbccEnd        = [dbadatetime]$endDbcc`
`DbccElapsed    = $dbccElapsed`

Reply:
Yep. Not sure it is worth anything but I'd use it over time if I was logging the activity and output of the process. Prevents the need of having to parse the command.

Reply:
Done